snippets/on.md

```diff
@@ -0,0 +1,25 @@
+### on
+
+Adds an event listener to an element with the ability to use event delegation.
+
+Use `EventTarget.addEventListener()` to add an event listener to an element. If there is a
+`target` property supplied to the options object, ensure the event target matches the
+target specified and then invoke the callback by supplying the correct `this` context.
+Use `options` to pass options to `addEventListener` or omit it to use bubbling by default.
+In order to use this function with `off`, the reference to the custom delegator function
+is returned if the `target` option is specified.
+
```

Comment:
Add *Omit `opts` to use a default value of `{}`* and explain what difference it makes.